Lib/ssl.py

```diff
@@ -531,7 +489,7 @@ def create_default_context(purpose=Purpose.SERVER_AUTH, *, cafile=None,
         context.verify_mode = CERT_REQUIRED
         context.check_hostname = True
     elif purpose == Purpose.CLIENT_AUTH:
-        context.set_ciphers(_RESTRICTED_SERVER_CIPHERS)
+        pass
```

Comment:
You can just remove this `elif` block entirely :-)

Reply:
Done